### ISSUE TITLE
Geelvink Bay > East Cenderawasih Bay; Bauzi reference categorisation

### ISSUE THREAD

**Issue by lauramarnold** [glottolog/glottolog#1121]
Hi Glottlog folks,

Can we update the family name 'Geelvink Bay' > 'East Cenderawasih Bay'? Following Foley (2018: 515):

"In previous literature and in Lewis et al. (2014), this family has been called the East Geelvink Bay family, but it strikes me as quite odd that, if we are to name a language family by a geographical feature of the area in which its languages are spoken, that we use a superceded name for that geographical feature to do so. Geelvink Bay is the old colonial Dutch name for what is now called Cenderawasih Bay. In keeping with this name change I have so named the family."

Anecdotally, others working in the region are also adopting this convention. 

I'm also wondering why Briley 1997 [https://glottolog.org/resource/reference/id/89891] is categorised as 'grammar', and Briley 1982 [https://glottolog.org/resource/reference/id/122775] is categorised as 'grammar_sketch'. Following the descriptors in Hammarström & Nordhoff (2012: 17), I would have thought they're both better characterised as 'specific_feature'.

**Comment by d97hah** [comment]
I've retained the name Geelvink Bay because (i) the alternative suggestion
Cenderawasih Bay is no less colonial than Geelvink Bay, i.e., it's the name
imposed by outsiders to this area, unlike e.g. Wissel Lakes -> Paniai
Lakes, (ii) Cenderawasih Bay is also/already used for the SHWNG subgroup,
and although this can be changed, this makes it all more confusable. Btw
who are the others working in the region? I did not know anyone was working
in this region and e.g. Donohue and Gil use (East) Geelvink Bay the last
time they wrote anything referring to this family.

Briley 1997, despite its title, is very extensive, so at least grammar
sketch, but you are right that 'grammar' is debatable. Looking at it again,
there not that much one misses but e.g. more extensive discussion of the
pluractional system would be expected in a grammar. Briley 1982 is
automatically tagged (and so marked) but I'll
fix this particular reference for next version.

all the best, H




Pada Rab, 18 Des 2024 pukul 23.25 lauramarnold ***@***.***>
menulis:

**Comment by lauramarnold** [comment]
Thanks for your quick reply, Harald. In response to (i), I read Foley's point about the colonial provenance as secondary to his point about 'Geelvink' having been superceded. You can remove the word 'colonial' from the quote, and his argument still stands. Given the choice between the two geographic terms, I think the one currently in use is preferable, as it's more transparent. I hadn't considered (ii), but it's a fair point. With the modifier 'East', I wonder how much of an issue it is in practice (cf. East Makian vs West Makian).   

> Btw who are the others working in the region? I did not know anyone was working in this region and e.g. Donohue and Gil use (East) Geelvink Bay the last time they wrote anything referring to this family.

By 'anecdotally' I meant in conversation only, not published materials. So the usual suspects working in NW New Guinea, to the extent that any of us talk about the family. I have a couple of overview papers in press that use East Cenderawasih Bay, with Geelvink in parentheses (happy to send proofs over). Fwiw, this convention is making its way into Wikipedia pages (e.g. [here](https://en.wikipedia.org/wiki/Papuan_languages)), and in [this blog post](https://thelanguagecloset.com/2024/09/14/who-influences-who-exploring-linguistic-influences-in-melanesia-through-the-numeral-lens/) it's East Cenderwasih (without the Geelvink). 

> Briley 1997, despite its title, is very extensive, so at least grammar sketch, but you are right that 'grammar' is debatable. Looking at it again, there not that much one misses but e.g. more extensive discussion of the pluractional system would be expected in a grammar. Briley 1982 is automatically tagged (and so marked) but I'll fix this particular reference for next version.

Cool, cheers!

**Comment by d97hah** [comment]
Hi Laura! Without the 'colonial' (and the implication that this would
be offensive) Foley's principle is not a good one. It's a bad idea to
rename established (sub-)family names because the name of geographic
inspiration has been superseded. It would just multiply the sea of names
and create more confusion. Fortunately, it's rarely done (e.g. we don't
have Tibeto-Myanmaric etc --- the examples are many). It could make sense
only if the previous name was only weakly established or the like. In the
case of Geelvink, it's as you know dominant throughout all literature
except Foley. The last person (afaik) to publish original research (Darsih
2012) on any of the lgs in question uses Geelvink and the later referential
work tends to use both (one in parentheses). So when/if the new name is
established in the literature I'll change Glottolog. The analogy with Makian
is not ideal as Makian is a language, and languages often have names
'in nature', whereas (sub-)families rarely do. It might work with East/West
or Makian Dalam etc but note how many authors feel the need to point out
that X Makian shouldn't be confused in with Y Makian.

Pada Sen, 23 Des 2024 pukul 04.35 lauramarnold ***@***.***>
menulis:

**Comment by lauramarnold** [comment]
Fair enough—thanks for taking the time to engage and explain the rationale, I appreciate it.

Happy holidays!

– Laura
